plugins/webapi/docs/api-reference.md

```diff
@@ -0,0 +1,1053 @@
+---
+title: GoShimmer API
+language_tabs:
+  - go: Go
+  - shell: Shell
+  - javascript--nodejs: Node.JS
+  - python: Python
+toc_footers: []
+includes: []
+search: true
+highlight_theme: darkula
+headingLevel: 2
+
+---
+
+<h1 id="goshimmer-api">GoShimmer API v0.1.0</h1>
+
+The GoShimmer API provides a simple and consistent way to get transactions from the Tangle, get a node's neighbors, or send new transactions.<br></br>This API accepts HTTP requests and responds with JSON data.
+
+## Base URLs
+
+All requests to this API should be prefixed with the following URL:
+
+```
+<a href="http://localhost:8080">http://localhost:8080</a>
+```
+
+<h1 id="transactions"></h1>
+
+## POST /broadcastData
+
+Creates a zero-value transaction and attaches it to the Tangle.
+
+Creates a zero-value transaction that includes the given data in the `signatureMessageFragment` field and the given address in the `address` field.<br></br>This endpoint also does tip selection and proof of work before attaching the transaction to the Tangle.
+
+### Body parameters
+
+```json
+{
+  "address": "string",
+  "data": "string"
+}
+```
+
+<h3 id="post__broadcastdata-parameters">Body parameters</h3>
+
+|**Name**|**Type**|**Required**|**Description**|
+|---|---|---|---|---|
+|body|object|true|Request object|
```

Comment:
this looks weird in rendered form, is this wanted?

Reply:
Looks like there's one too many columns

Reply:
Fixed them 👍

Reply:
cool thanks